src/SeleniumLibrary/keywords/javascript.py

```diff
@@ -77,15 +77,64 @@ def execute_async_javascript(self, *code):
         self.info("Executing Asynchronous JavaScript:\n%s" % js)
         return self.driver.execute_async_script(js)
 
+    @keyword
+    def execute_javascript_with_arguments(self, *code):
+        """ Executes the given JavaScript code with the given arguments
+        #TODO: Add docstring
+        """
+
+        code, args = self._parse_javascript_and_arguments(code)
+        js = self._get_javascript_to_execute(code)
+
+        if len(args) == 0:
+            self.info("Executing JavaScript \n%s" % (
+                js))
+            return self.driver.execute_script(js)
+        else:
+            self.info("Executing JavaScript \n%s\nwith arguments \n%s" % (
+                code, args))
+            return self.driver.execute_script(code, args)
```

Comment:
By reading the code, which makes it a guess, I think the problem is in this line. You should unpack the `args` and not pass it as a list. So instead `(code, args)` write `(code, *args)`.